Makefile

```diff
@@ -197,7 +197,7 @@ $(BINDIR):
 
 .scBuildImage: build/build-image/Dockerfile
 	sed "s/GO_VERSION/$(GO_VERSION)/g" < build/build-image/Dockerfile | \
-	  docker build -t scbuildimage -
+	  docker build -t scbuildimage -f - .
```

Comment:
I'm shocked this used to work.

Reply:
So what we had before worked but it didn't send any build context (i.e. the files in our repo). I've changed it so that it reads the dockerfile from stdin like before but also uses the current directory for build context. Then used a .dockerignore to ensure that only the vendor files we need are included.